iron-dropdown.html

```diff
@@ -239,7 +240,7 @@
           this.$.contentWrapper.classList.remove('animating');
           if (this.opened) {
             Polymer.IronOverlayBehaviorImpl._renderOpened.apply(this);
-          } else if (!this.cancelled) {
+          } else if (!this.canceled) {
```

Comment:
Yeah. Appeasing the compiler can be a pain in the ass, but it's nice when it catches this sort of thing.